numpy/distutils/tests/test_build_ext.py

```diff
@@ -7,7 +7,10 @@
 import pytest
 from numpy.testing import IS_WASM
 
-@pytest.mark.skipif(IS_WASM, reason="cannot start subprocess in wasm")
+@pytest.mark.skipif(
+    IS_WASM or (sys.maxsize < 2 ** 31 + 1),
+    reason="Cannot start subprocess in WASM or on 32-bit",
+)
```

Comment:
Actually looking at the test, it can be removed completely, it is essentially testing `distutils` functionality. If `meson` has a problem with static library linkage it wouldn't be our issue AFAIK.

Reply:
Actually, @mattip, this is because on the [Azure run with 3.9](https://dev.azure.com/numpy/numpy/_build/results?buildId=33191&view=logs&j=94356751-2a09-5f35-07c1-87bf12815f8e&t=aed0f7cb-c656-5ca0-8fcd-90f7c267ae7c) the `distutils` test fails since it lacks `gfortran`.. This is because the testing utility it imports no longer exists, so I will `xfail` the test instead.